packages/core/test/insured-bridge/BridgePool.js

```diff
@@ -1,6 +1,6 @@
 const { assert } = require("chai");
 const hre = require("hardhat");
-const { web3 } = require("hardhat");
+const { web3 } = hre;
```

Comment:
I think we do this in a few places wrong! I'll send out a quick PR to update the repo for anywhere we do this incorrectly.

Reply:
PR: https://github.com/UMAprotocol/protocol/pull/3418